tests/testthat/test-quo.R

```diff
@@ -134,9 +135,9 @@ test_that("replace_values_by_names Test 11: names of argument is NULL", {
 
 ## Test 12: warning if quosures argument is used ----
 test_that("replace_values_by_names Test 12: warning if quosures argument is used", {
-  expect_warning(
+  expect_error(
     replace_values_by_names(quosures = rlang::quos(STUDYID, USUBJID)),
-    class = "lifecycle_warning_deprecated"
+    class = "lifecycle_error_deprecated"
   )
 })
```

Comment:
can you check that these don't admit error/warnings when installing, e.g. #268 .  I think we can suppress them??

Reply:
Looks resolved at least on my end!

Reply:
fabulous